src/io/dirops.c

```diff
@@ -66,6 +66,18 @@ void MVM_dir_rmdir(MVMThreadContext *tc, MVMString *path) {
     char * const pathname = MVM_platform_path(tc, path);
     uv_fs_t req;
     int rmdir_error = uv_fs_rmdir(NULL, &req, pathname, NULL);
+#ifdef _WIN32
+    if (rmdir_error == ENOTEMPTY) {
```

Comment:
It probably doesn't matter, but should the errors check for e.g. `UV_ENOTEMPTY`?